src/common/open-api/services/HumanTaskService.ts

```diff
@@ -125,6 +125,7 @@ export class HumanTaskService {
   public claimTask(
     taskId: string,
     overrideAssignment: boolean = false,
+    withTemplate: boolean = true,
```

Comment:
I think this one is false by default in conductor. hence it should be false here